includes/google_scholar.inc

```diff
@@ -171,7 +178,7 @@ function islandora_scholar_create_meta_tags($object) {
 
     $online_date = $mods_xml->xpath(variable_get('islandora_scholar_xpaths_online_date', '//mods:recordInfo/mods:recordCreationDate'));
     if ($online_date) {
-      $date_string = islandora_scholar_parse_date_foryear($online_date);
+      $date_string =  islandora_parse_date_foryear($online_date);
```

Comment:
This change breaks the page load for me; the function `islandora_parse_date_foryear` does not exist in any of the Islandora modules. `islandora_scholar_parse_date_foryear` does exist later on in this same file though at https://github.com/Islandora/islandora_scholar/blob/7.x/includes/google_scholar.inc#L210-L229.

I'm assuming this is just a typo, but this needs to be changed back to the way it was originally before this PR can be accepted.

Reply:
I am not sure how it's breaking the page load. The it was calling for the function in the code. If changing it to islandora_parse_date_foryear, I will also need to change the name for the function down the line.

Reply:
yep, it is a typo. Fixing it